scripts/install.sh

```diff
@@ -64,14 +64,13 @@ sudo cp conf/tomcat.service /etc/systemd/system/
 sudo cp conf/tomcat-users.xml /opt/tomcat/conf/tomcat-users.xml
 sudo chown tomcat:tomcat /opt/tomcat/conf/tomcat-users.xml
 
-#make tomcat a part of start-up scripts
-sudo update-rc.d tomcat defaults
-sudo update-rc.d tomcat enable
-
 #reload the systemd daemon so that it knows about our service file
 sudo systemctl daemon-reload
 sudo systemctl start tomcat
 
+#make tomcat a part of start-up scripts
+sudo systemctl enable tomcat.service
```

Comment:
shouldn't this come before the command on line-68?

Reply:
I think it should come after so as to let the system first know about the presence of a new file inside /etc/systemd/system